build.xml

```diff
@@ -453,7 +453,7 @@ To get started using Eclipse, execute "ant build-dev" and import the top-level
     </target>
 
     <target name="release-docs" depends="release-javadoc,release-sphinx-api,release-slice2html" description="Generate Docs for all components under dist/docs/api">
-        <zip destfile="${omero.home}/target/OMERO.docs-${omero.version}.zip">
+        <zip destfile="${omero.home}/target/OMERO.javadoc-${omero.version}.zip">
             <zipfileset dir="${dist.dir}/docs" prefix="OMERO.docs-${omero.version}"/>
```

Comment:
I'm honestly not sure! I ummed and ahhed about changing that too but given I can't test locally, I've failed to work out if I need to